<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorWidget</class>
 <widget class="QWidget" name="SensorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Label_SensorID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>传感器ID: 000000</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_SensorType">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>类型: fNIRS</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Battery">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>电量: 0%</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ComboBox_SampleRate">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>0</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::NoInsert</enum>
   </property>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Label_SampleRateText">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>采样率</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_SampleRate">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>0</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Channel">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>通道配置</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SpinBox_Lights">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>40</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
   <property name="value">
    <number>8</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="SpinBox_Detectors">
   <property name="geometry">
    <rect>
     <x>1440</x>
     <y>40</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
   <property name="value">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLabel" name="Label_LightsText">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>光源</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_DetectorsText">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>探测器</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="ScrollArea_Channels">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>70</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="Widget_Channels">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>229</width>
      <height>209</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="GridLayout_Channels"/>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="Button_Channels">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>0</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="ScrollArea_Plot">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>1221</width>
     <height>261</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="Widget_Plot">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1219</width>
      <height>259</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>1221</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="HLayout_Plot"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
